.github/workflows/package-apisix-rpm-ubi.yml

```diff
@@ -0,0 +1,105 @@
+          sudo gem install --no-document fpm
+          sudo apt-get install -y rpm
+
+      - name: packaging APISIX(-remote) with remote code
+        run: |
+          make package type=rpm app=apisix version=master checkout=master image_base=registry.access.redhat.com/ubi8/ubi image_tag=8.6 artifact=apisix-remote
+
+      - name: packaging APISIX(-local) with local code
+        run: |
+          git clone https://github.com/apache/apisix.git
+          make package type=rpm app=apisix version=master checkout=master image_base=registry.access.redhat.com/ubi8/ubi image_tag=8.6 local_code_path=./apisix artifact=apisix-local
+
+      - name: run ubi8 docker and mapping rpm into container
+        run: |
+          docker run -itd -v $PWD/output:/output --name ubiInstance --net="host" registry.access.redhat.com/ubi8/ubi:8.6 /bin/bash
+
+      - name: enable apisix repository in container
+        run: |
+          docker exec ubiInstance bash -c "rpm --import https://repos.apiseven.com/KEYS"
+          docker exec ubiInstance bash -c "yum -y install https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm"
+
+      - name: install APISIX(-remote) master by rpm in container
+        run: |
+          docker exec ubiInstance bash -c "yum -y localinstall /output/apisix-remote-master-0.ubi8.6.x86_64.rpm"
+          docker exec ubiInstance bash -c "apisix start"
+
+      - name: check and ensure APISIX(-remote) master is installed
+        run: |
+          code=$(curl -k -i -m 20 -o /dev/null -s -w %{http_code} http://127.0.0.1:9180/apisix/admin/routes -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1')
+          if [ ! $code -eq 200 ]; then
+              echo "failed: failed to install Apache APISIX by rpm"
+              exit 1
+          fi
+
+      - name: stop and uninstall APISIX(-remote) master
+        run: |
+          docker exec ubiInstance bash -c 'PIDS=$(pgrep -f nginx); for PID in $PIDS; do kill -TERM $PID; done'
+          docker exec ubiInstance bash -c "yum -y erase apisix-remote-master"
+
+      - name: install APISIX(-remote) master by rpm in container
+        run: |
+          docker exec ubiInstance bash -c "yum -y localinstall /output/apisix-remote-master-0.ubi8.6.x86_64.rpm"
+          docker exec ubiInstance bash -c "apisix start"
+
+      - name: check and ensure APISIX(-remote) master is installed
+        run: |
+          code=$(curl -k -i -m 20 -o /dev/null -s -w %{http_code} http://127.0.0.1:9180/apisix/admin/routes -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1')
+          if [ ! $code -eq 200 ]; then
+              echo "failed: failed to install Apache APISIX by rpm"
+              exit 1
+          fi
+
+      - name: stop and uninstall APISIX(-remote) master
+        run: |
+          docker exec ubiInstance bash -c 'PIDS=$(pgrep -f nginx); for PID in $PIDS; do kill -TERM $PID; done'
+          docker exec ubiInstance bash -c "yum -y erase apisix-remote-master"
```

Comment:
This part seems to be repeated from the above section, right?   L58～69

Reply:
@tao12345666333  You're right. And this duplication is in other workflows as well. Should I remove this duplication of 3 steps from other workflows as well?

Reply:
Removed duplicate steps